includes/class-utilities.php

```diff
@@ -11,17 +11,14 @@
  * The Class for Admin Settings Page and functions to access Settings Values.
  */
 class Utilities {
-
 	/**
-	 * Get the top level domain name from the site URL.
+	 * Get the domain name from the site URL.
 	 *
-	 * @return string the top level domain name.
+	 * @return string the domain name.
```

Comment:
```suggestion
	 * @return string The domain name.
```
Nit: Capitalize the first character.